packages/website/pages/core.md

```diff
@@ -245,3 +245,58 @@ defineComponent({
     shadowMode: 'open',
 });
 ```
+
+## Components
```

Comment:
we'll want to call out that all of these elements will add a wrapper element to the component definition, but not to the resultant dom.